src/renderer/services/labkey-client/index.ts

```diff
@@ -306,6 +306,25 @@ export default class LabkeyClient extends HttpCacheClient {
     return response.rows;
   }
 
+  /**
+   * Get all options for program from labkey
+   */
+  public async getProgramOptions(): Promise<AnnotationOption[]> {
+    const filters = ["query.AnnotationId~eq=153"];
```

Comment:
Looks like this replaces the previous filtering that you did in `index`, and then passes the result to there via an established redux pattern (which i can't follow for the life of me) ?

Reply:
> previously the options came with the template via annotationOptions

Huh, interesting. There are currently only 645 "AnnotationOption" rows -> In other places, do we just filter on those when we need them? Seems a little clunky.

Nitpicky, but I'd probably take out the `153` on this line and replace it with a `PROGRAM_ANNOTATION_ID` constant, just to make the query a bit more obvious.